_plugins/fenl_catalog.rb

```diff
@@ -0,0 +1,15 @@
+module Jekyll
+    class FenlCatalog < Liquid::Tag
+
+      def initialize(tag_name, function, tokens)
+        super
+        @function = function.strip
+      end
+
+      def render(context)
+        "https://kaskada.io/docs-site/kaskada/main/fenl/catalog.html##{@function}"
+      end
+    end
+  end
+
+  Liquid::Template.register_tag('fenl_catalog', Jekyll::FenlCatalog)
```

Comment:
Introduced a `fenl_catalog` plugin so we can do `{% fenl_catalog min %}` and get a link to the corresponding fenl catalog page. I figure this lets us change one place if we split the catalog into separate pages, etc.